docs/discussion-topics/c-wallet-unit-attestation.md

```diff
@@ -641,7 +641,6 @@ R14, SR1, TR39, TR84, TR85 are particularly relevant to consider given the discu
 
 | Reference       | Description                                                                                                                                                                                                                                                                          |
 |-----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [WebAuthN]      | Web Authentication: An API for accessing Public Key Credentials Level 2 W3C Recommendation, 8 April 2021, https://www.w3.org/TR/webauthn-2/                                                                                                                                          |
```

Comment:
In line 253 this PR introduces the term Webauthn. Doesn't this mean that we need the reference?

Reply:
You mean [here](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/pull/400/files#diff-0a839b15983a4fffd02a881922f4789a5957d556d844a6cf432e2f921d9b8d4aR235)? That's in a different file. This file has no uses of the `[WebAuthN]` reference (or any other upper/lowercase spelling of "webauthn").

The other file already references "WebAuthN" on [line 180](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/pull/400/files#diff-0a839b15983a4fffd02a881922f4789a5957d556d844a6cf432e2f921d9b8d4aL180) without a reference, so this PR doesn't introduce a new term to the file. But we can of course add a reference there if you'd like.

Reply:
You are correct. We can take out this reference here